sources/ftypes.h

```diff
@@ -323,6 +323,18 @@ typedef int (*TFUN1)();
 #define SORTPOWERFIRST 2
 #define SORTANTIPOWER 3
 
+/* These control the output of WriteStats. CHECKLOGTYPE implies
```

Comment:
It is more natural to have constants first for `par` and then for `checkLogType`.

Reply:
Thanks for the review. This is resolved in [c7b0a51](https://github.com/vermaseren/form/pull/493/commits/c7b0a51356a81b87180d945ffae3f07c720be7e6)